docs/source/AdministratorGuide/Tutorials/installDFC.rst

```diff
@@ -0,0 +1,130 @@
+.. _tuto_install_dfc:
+
+=================================
+Installing the DIRAC File Catalog
+=================================
+
+Pre-requisite
+=============
+
+You should:
+
+* have a machine setup as described in :ref:`tuto_basic_setup`
+* be able to install dirac components
+* have installed a DIRAC SE using the tutorial (:ref:`tuto_install_dirac_se`).
+
+Tutorial goal
+=============
+
+The aim of the tutorial is to install the DIRAC FileCatalog (DFC)
+By the end of the tutorial, you will be able to do all sort of simple Data Management operations.
+
+More links
+==========
+
+More information can be found at the following places:
+
+* Introduction to DataManagement: :ref:`data-management-system`
+* Catalog resource definition :ref:`resourcesCatalog`
+* How-to datamanagement for user :ref:`howto_user_dms`
+
+Installing the DFC
+==================
+
+This section is to be executed as ``diracuser`` with a proxy with ``dirac_admin`` group.
+
+The DFC is no different than any other DIRAC service with a database. The installation step are thus very simple::
+
+  [diracuser@dirac-tuto ~]$ dirac-admin-sysadmin-cli --host dirac-tuto
+  Pinging dirac-tuto...
+  [dirac-tuto]> install db FileCatalogDB
+  Adding to CS DataManagement/FileCatalogDB
+  Database FileCatalogDB from DIRAC/DataManagementSystem installed successfully
+  [dirac-tuto]> install service DataManagement FileCatalog
+  Loading configuration template /home/diracuser/DIRAC/DIRAC/DataManagementSystem/ConfigTemplate.cfg
+  Adding to CS service DataManagement/FileCatalog
+  service DataManagement_FileCatalog is installed, runit status: Run
+
+
+Adding the FileCatalog resource
+===============================
+
+In order to be used as a FileCatalog by clients, the DFC needs to be declared. This happens in two places:
+
+* ``/Resources/FileCatalogs/``: in this section, you define how to access the catalog
+* ``/Operations/Defaults/Services/Catalogs/``: in this section, you define how to use the catalog (for example read/write)
+
+
+Since we have only one catalog, we will use it as ``Read-Write`` and as ``Master``.
+
+Using the WebApp, add the following in ``/Resources/FileCatalogs/`` (all options to defaults)::
```

Comment:
Mention the user/group you need to have.

Reply:
done